diff --git a/client/src/main.c b/client/src/main.c
index e016aca..abbd1df 100644
--- a/client/src/main.c
+++ b/client/src/main.c
@@ -1637,12 +1637,6 @@ static void lg_shutdown()
 
 int main(int argc, char * argv[])
 {
-  if (getuid() == 0)
-  {
-    DEBUG_ERROR("Do not run looking glass as root!");
-    return -1;
-  }
-
   DEBUG_INFO("Looking Glass (" BUILD_VERSION ")");
   DEBUG_INFO("Locking Method: " LG_LOCK_MODE);
 
